board/piksiv3/rootfs-overlay/etc/init.d/S83zmq_file_logger

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Are we sure this should be s83?  
What is our reasoning for booting this up before the upgrades script?  

We should probablly kill this process if an update starts in the upgrade.sh script.  I wouldn't want to see usb/io contention prevent the ability of an upgrade to proceed.